AppCreationScripts/apps.json

```diff
@@ -0,0 +1,44 @@
+{
+  /*
+  This section describes the Azure AD Applications to configure, and their dependencies
+  */
+
+  "Sample": {
+    "Title": "Signing users to an ASP.NET Core Web App",
+    "Level": 200,
+    "Client": "ASP.NET Core 2.2"
+  },
+  "AppRegistrations": [
+    {
+      "x-ms-id": "active-directory-aspnetcore-webapp-openidconnect-v2",
+      "x-ms-name": "aspnetcore-webapp-openidconnect-v2",
+      "x-ms-version": "2.0",
+      "replyUrlsWithType": [
+        {
+          "url": "https://localhost:44321/",
+          "type": "Web"
+        },
+        {
+          "url": "https://localhost:44321/signin-oidc",
+          "type": "Web"
+        }
+      ],
+      "oauth2AllowIdTokenImplicitFlow": true,
+      "oauth2AllowImplicitFlow": true,
```

Comment:
Wouldn't the IdToken the only one required?

Reply:
yes, @andretms (See the email thread with Adrian)